imagine/tests/test_observabledict.py

```diff
@@ -140,7 +137,7 @@ def test_covdict_apply_mask(self):
         covdict.append(('test', 'nan', str(2*mpisize), 'nan'), cov, True)
         covdict.apply_mask(mskdict)
         pix_num = msk.sum()
-        self.assertTrue(('test', 'nan', str(pix_num), 'nan') in covdict.keys())
+        assert ('test', 'nan', str(pix_num), 'nan') in covdict.keys()
 
 if __name__ == '__main__':
     unittest.main()
```

Comment:
Will this still work?

Reply:
Do you think I would push it if it didn't work properly?

Reply:
I was referring specifically to  the last lines:
```python
if __name__ == '__main__':
    unittest.main()
```
as thought `unittest` was removed from the module

Reply:
Uhm, that's weird.
Those lines are not supposed to be there.

Reply:
That's what code review is for!   : - )

Reply:
Well, it would still work, until you execute the file instead of importing it (which is what `pytest` does).